### PR TITLE
Update on `ignore()` method

### PR DESCRIPTION
**Pull request by vaish567**
Hello @ozziest I hope, this is correct😅.
- [x] Update on #32

### CODE REVIEW

**Comment by ozziest** [comment]
@vaish567 , thanks a lot. We are so close. I reviewed your code. But we don't need to create a new `Model` class. Can you add the `ignore` getter to the `Model` directly, please?

**Comment by vaish567** [comment]
Hey @ozziest I did changes in `model`, please check it and let me know.

File: src/resolvers/setExpressRoutes.js [hunk 1]
```diff
@@ -132,6 +132,10 @@ const createNestedRoutes = async (
     (relation) =>
       relation.model === model.name && relation.type === RELATIONSHIPS.HAS_MANY
   );
+  
+  if (model.instance.ignore) {
```
Comment (ozziest) [inline src/resolvers/setExpressRoutes.js:136]: One last thing, can you put the if block to the `createRouteByModel` function, please?

**Review by ozziest — CHANGES_REQUESTED** [review]

File: src/resolvers/setExpressRoutes.js (hunk 1 above)
Comment (vaish567) [reply src/resolvers/setExpressRoutes.js:136]: > One last thing, can you put the if block to the `createRouteByModel` function, please?

Surely :)

**Comment by vaish567** [comment]
Hello @ozziest Please check if this okay?

**Comment by vaish567** [comment]
Hey @ozziest I hope, it's right☺️.

**Comment by ozziest** [comment]
@vaish567 , congrats! 🥳 

We are so happy about your contribution.

**Comment by ozziest** [comment]
Documentation is live; https://axe-api.github.io/04-models/#ignore

**Comment by vaish567** [comment]
> Documentation is live; https://axe-api.github.io/04-models/#ignore

I'm so happy it's working finally :)

**Pull request merged**